components/Dialogs/src/DialogThemer/MDCAlertScheme.m

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCAlertScheme.h"
+
+static const CGFloat kCornerRadius = 4.0f;
+
+@implementation MDCAlertScheme
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    _colorScheme = [MDCSemanticColorScheme new];
+    _typographyScheme = [MDCTypographyScheme new];
```

Comment:
_Nit:_ Use `alloc] init]` instead of `new]`

Reply:
Is this a convention or a readability guideline? I see both new and alloc/init being used.

Reply:
Objective-C Style guide.

http://google.github.io/styleguide/objcguide.html#do-not-use-new